source/contracts/reporting/Market.sol

```diff
@@ -387,6 +391,10 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
         return finalPayoutDistributionHash;
     }
 
+    function getDesignatedReportPayoutHash() public constant returns (bytes32) {
+        return designatedReportPayoutHash;
+    }
```

Comment:
Unrelated to this PR, I wonder if manual getters are still fewer bytes than auto-getters with optimizations enabled?

Reply:
Will experiment at some point with that, although we're well within size limits now too